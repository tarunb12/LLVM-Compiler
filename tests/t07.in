int main() {
    int i = 6
    int j = 0
    while (j < i) {
        if (j == 4) break
        printf("%d\n", j++)
    }
    return 0
}