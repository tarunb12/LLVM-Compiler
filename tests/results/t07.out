1 iteration (no breaks yet)
2 iteration (no breaks yet)
3 iteration (no breaks yet)
4 iteration (no breaks yet)
